README.md

```diff
@@ -295,7 +295,7 @@ You should see outputs to stdout of the form --
 Environment: ipython
 Tools: brave_search, wolfram_alpha, photogen
 
-Cutting Knowledge Date: December 2023
+Knowledge Cutoff Date: December 2023
```

Comment:
Don't update this! It's the system prompt the model was trained on and should match.

Reply:
Let's go